contracts/starknet/lib/proposal.cairo

```diff
@@ -4,7 +4,6 @@ from starkware.cairo.common.uint256 import Uint256
 struct Proposal:
     member execution_hash : Uint256
     member start_timestamp : felt
-    member end_timestamp : felt
```

Comment:
I've removed that and just compute the end timestamp now. It's one write less :)

Reply:
But then if someone change period the proposal will also change period?

Reply:
Hmm you're right. Currently there's no way of editing those so I thought it wasn't an issue but maybe I should already write the code with this in mind, so that when the day comes and we decide to make those editable, we don't introduce any bugs :)

Reply:
339b15a11f2691287728e2d798c374af44a711e8

Reply:
Great